scripts/deploy_bug.py

```diff
@@ -66,57 +72,63 @@ def main(argv):
 
     # Convert to int and drop any that don't
     tags = [int(tag) for tag in tags if tag.isdigit()]
+    return tags
+
+
+def main(argv):
+    # Fetch tags as numbers (e.g. 309)
+    tags = get_tags()
 
     # The current tag is the max of that
     current_tag = max(tags)
 
     # The next tag is current_tag + 1
     next_tag = current_tag + 1
 
+    # Figure out the current version
+    sha = fetch_current_revision(HOST)
+
+    # Get the commits between the currently deployed version and what's in
+    # master tip and if there's nothing to deploy, then we're done!
+    resp = fetch_history_from_github('mozilla-services', 'socorro', sha)
+    if resp['status'] != 'ahead':
+        print('Nothing to deploy!')
+        return
+
+    commits = resp['commits']
+
     print('summary:')
     print('socorro deploy: %s' % next_tag)
     print()
-
     print('description:')
     print()
-
     print('We want to do a Socorro -prod deploy today tagged %s.' % next_tag)
     print()
-
     print('It consists of the following:')
     print()
+    print('(current tag: %s - %s)' % (current_tag, sha[:8]))
 
-    # Figure out and print the version
+    # Print the commits out skipping merge commits
+    for commit in commits:
+        if len(commit['parents']) > 1:
+            continue
```

Comment:
Listing the commits continues to omit merge commits. This makes it muuuuuch easier to read through the list of things going out and know what we need to do pre/post deploy. Otherwise it's messy.

Before, we removed the merge commits after getting all the commits. Now we just ignore them when listing the commits, but when figuring out the commit to tag, we don't ignore merge commits.

Reply:
Makes sense to me.